packages/core/src/Stores/notification-store.js

```diff
@@ -890,6 +905,18 @@ export default class NotificationStore extends BaseStore {
                 ),
                 type: 'warning',
             },
+            poinc_upload_limited: {
+                key: 'poinc_upload_limited',
+                header: localize("You've reached the limit of uploading your documents."),
+                message: localize('Please contact us via live chat.'),
+                action: {
+                    onClick: () => {
+                        window.LC_API.open_chat_window();
+                    },
+                    text: localize('Go to live chat'),
+                },
+                type: 'danger',
+            },
```

Comment:
Just a note, Should'nt we specify to client for which document he has reached the limit?  POI, POA or Proof of income?

Reply:
sounds reasonable, but according to design we have no mentioning  about type of document in this notification